src/plugins/management/public/plugin.ts

```diff
@@ -107,10 +110,15 @@ export class ManagementPlugin
         const { renderApp } = await import('./application');
         const [coreStart] = await core.getStartServices();
 
+        coreStart.chrome
+          .getChromeStyle$()
+          .subscribe((style) => (managementPlugin.isSidebarEnabled = style !== 'project'));
```

Comment:
This probably creates a memory leak because the mount is called every time when app mounts and there is no unsubscribe here. 

I think it would be easier to provide getIsSidebarEnabled$ observable inside the app

Reply:
Awesome catch, thank you!